elf/Makefile

```diff
@@ -2,7 +2,7 @@ all: prog.uelf.o
 
 include ../Makefile.common
 
-CFLAGS   = -std=gnu11 -O0 -Wall -Werror -fno-builtin -ffreestanding
+CFLAGS   = -std=gnu11 -Og -Wall -Werror -fno-builtin -ffreestanding
```

Comment:
It has not improved much, apparently, which is why I keep flipping between `-O0` and `-Og`. Debugging the loaded code is tricky, as the gdb can't access debugging information there - so I'm mostly stuck with `objdump -d -S prog.o`. `-Og` tends to do pretty confusing tricks at asm level, while `-O0` usually generates fat too much instructions to step through them conveniently.